modules/install/src/test/scala/coursier/install/InstallTests.scala

```diff
@@ -835,12 +839,14 @@ object InstallTests extends TestSuite {
             "scala.usejavacp=true",
             "scala-compiler.version=2.12.8"
           )
+
         testRun(scala2CompilerJars, scala2Properties)
 
         val updated = installDir0.createOrUpdate(appInfo0)
         assert(updated.exists(identity))
 
-        testRun(scala3CompilerJars, scala3Properties)
```

Comment:
Can't we keep this check (`testRun`), alongside the new `testOutput`?

Reply:
so I removed this one because `testRun` is specifically looking for files inside a bootstrapped launcher jar, but with the change in the test it is now a prebuilt launcher, so these entries don't exist

Reply:
maybe I can do a check that the file exists and matches some shape?

Reply:
I added a test that searches for the path to the launcher in the script file